dashboard/lib/services/curriculum_pdfs/resources.rb

```diff
@@ -83,7 +83,7 @@ def generate_script_resources_pdf(script, directory = "/tmp/")
             PDF.merge_local_pdfs(destination, *pdfs)
           rescue Exception => exception
             ChatClient.log(
-              "Error when trying to merge resource PDFs for #{script.name}: #{exception}",
+              "@teacher-tools-team Error when trying to merge resource PDFs for #{script.name}: #{exception}",
```

Comment:
@kobryan0619 , would this failure block build in staging. Asking as I'm wondering if there is value in tagging teacher tools here.

Reply:
I think there is value in it.  Also, Darin pointed me to [this](https://github.com/code-dot-org/infrastructure/blob/main/scripts/oncall_sync.py) script which we could add to for this if we wanted to focus the tagging for the person on call (and also move our on call schedule to PagerDuty)

Reply:
Thanks! I'll make this as a follow up change.